spec/core_functions/color/mix/both_weights.hrx

```diff
@@ -0,0 +1,70 @@
+<===> transparent/first/input.scss
+a {b: mix(transparent, #0144bf, 70%)}
+
+<===> transparent/first/output.css
+a {
+  b: rgba(1, 68, 191, 0.3);
+}
+
+<===>
+================================================================================
+<===> transparent/last/input.scss
+a {b: mix(#91e16f, transparent, 70%)}
+
+<===> transparent/last/output.css
+a {
+  b: rgba(145, 225, 111, 0.7);
+}
+
+<===>
+================================================================================
+<===> weighted/first/input.scss
+a {b: mix(rgba(#91e16f, 0.2), rgba(#0144bf, 0.7), 100%)}
+
+<===> weighted/first/output.css
+a {
+  b: rgba(145, 225, 111, 0.2);
+}
+
+<===>
+================================================================================
+<===> weighted/last/input.scss
+a {b: mix(rgba(#91e16f, 0.2), rgba(#0144bf, 0.7), 0%)}
+
+<===> weighted/last/output.css
+a {
+  b: rgba(1, 68, 191, 0.7);
+}
+
+<===>
+================================================================================
+<===> mixed/firstwards/input.scss
+a {b: mix(rgba(#91e16f, 0.8), rgba(#0144bf, 0.3), 63%)}
+
+<===> mixed/firstwards/output.css
+a {
+  b: rgba(121, 199, 124, 0.615);
+}
+
+<===>
+================================================================================
+<===> mixed/lastwards/input.scss
+a {b: mix(rgba(#91e16f, 0.2), rgba(#0144bf, 0.7), 42%)}
+
+<===> mixed/lastwards/output.css
+a {
+  b: rgba(29, 99, 175, 0.49);
+}
+
+<===>
+================================================================================
+<===> contradiction/input.scss
+// When we weight entirely towards a transparent color, the formula for
+// computing the combined alpha would divide by zero, so we just return
+// transparent as a special case.
+a {b: mix(transparent, #0144bf, 100%)}
+
+<===> contradiction/output.css
+a {
+  b: rgba(0, 0, 0, 0);
```

Comment:
I think this was changed back to `rgba(0, 0, 0, 0)`. Should it still be `transparent`? @nex3

Reply:
`rgba(0, 0, 0, 0)` matches the existing output here. If I recall, we chose to emit this instead of `transparent` because that keyword had some weird behavior in very old IE versions.